.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
+  <state>
+    <option name="USE_PER_PROJECT_SETTINGS" value="true" />
+  </state>
+</component>
```

Comment:
- [ ] add new line

Reply:
The config files are automatically created by the IDE tool. I don't want to edit these files afterwards.